include/deal.II/lac/constraint_matrix.h

```diff
@@ -1361,6 +1374,8 @@ void
 ConstraintMatrix::add_line (const size_type line)
 {
   Assert (sorted==false, ExcMatrixIsClosed());
+  Assert (!local_lines.size() || local_lines.is_element(line),
```

Comment:
not really needed, because we will do this check in calculate_line_index() a few lines below.